src/meson.build

```
@@ -27,7 +28,8 @@ vala_files = [
     'Widgets/PartitionBar.vala',
     'Widgets/PartitionMenu.vala',
     'Widgets/Terminal.vala',
-    'Widgets/VariantWidget.vala'
+    'Widgets/VariantWidget.vala',
+    join_paths(meson.source_root(), 'common', 'DBusStructures.vala'),
```

```suggestion
    join_paths('..', 'common', 'DBusStructures.vala'),
```
`meson.source_root()` is deprecated